EasyInventory/src/main/resources/templates/inventory.html

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
-<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<html xmlns:th="http://www.thymeleaf.org" lang="">
```

Comment:
Fixed the URL since it should be referencing thymeleaf

Reply:
yes.